srcalc/src/main/java/gov/va/med/srcalc/domain/Patient.java

```diff
@@ -1,9 +1,12 @@
 package gov.va.med.srcalc.domain;
 
 import gov.va.med.srcalc.domain.calculation.RetrievedValue;
+import gov.va.med.srcalc.vista.HealthFactor;
```

Comment:
The domain model should not depend on any other packages. Just because something comes from VistA doesn't mean it needs to be in the `vista` package. (See the package Javadoc.) I think the `HealthFactor` class could move to the domain model.